src/ui/pages/Credentials/Credentials.tsx

```diff
@@ -82,6 +96,15 @@ const Credentials = () => {
   const [deletedPendingItem, setDeletePendingItem] =
     useState<CredentialShortDetails | null>(null);
   const [openDeletePendingAlert, setOpenDeletePendingAlert] = useState(false);
+  const [individualCredentials, setIndividualCredentials] = useState<
+    CredentialShortDetails[]
+  >([]);
+  const [groupCredentials, setGroupCredentials] = useState<
+    CredentialShortDetails[]
+  >([]);
+  const [selectedFilter, setSelectedFilter] = useState<CredentialsFilters>(
```

Comment:
I don't think we need react state here. Can we use redux state ?

`const selectedFilter = credentialsFiltersCache ?? CredentialsFilters.All;`